### DISCUSSION TITLE
Geomap: Location mode based on IP location

### DISCUSSION THREAD

**Discussion by marcoboffi** [grafana/grafana#68241] [Dashboards, Panels & Visualizations]
I suggest to add a new 'location mode' to Geomap panel based on location of an IPv4 field specified during panel configuration.  
It's possible to find IP address using geolocation API or database.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82676. Feel free to continue the discussion around this there. Thank you!
